src/Dependencies/Collections/ImmutableSegmentedDictionary`2+PrivateInterlocked.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Collections
+{
+    internal readonly partial struct ImmutableSegmentedDictionary<TKey, TValue>
+    {
+        internal static class PrivateInterlocked
+        {
+            internal static ImmutableSegmentedDictionary<TKey, TValue> VolatileRead(ref ImmutableSegmentedDictionary<TKey, TValue> location)
```

Comment:
>ref [](start = 84, length = 3)

Can this be `in`? The location is not modified, right?

Reply:
Yes, we can't change `Volatile.Read` but we can at least fix this one.

Reply:
➡️ This is now fixed